pkgs/by-name/sw/swayfx-unwrapped/package.nix

```diff
@@ -18,6 +19,8 @@
     systemdSupport
     trayEnabled
     ;
+
+  wlroots_0_17 = wlroots_0_16;
```

Comment:
:cry: maybe we should make it `wlroots` there after all then

Reply:
Neither is great, but passing `wlroots_0_16` for `wlroots_0_17` seems rather confusing to me...

Reply:
I think this ends up being the limitation of the current `pkgs/by-name` approach.

But really, I think we should just decouple `swayfx` from `sway` derivation instead. Copying-and-paste sometimes is better than having a bad code reuse, and the fact that any change in `sway` can break `swayfx` is a strong argument to not having those 2 derivations share code.